hscontrol/policy/acls.go

```diff
@@ -108,35 +107,22 @@ func LoadACLPolicyFromPath(path string) (*ACLPolicy, error) {
 		Bytes("file", policyBytes).
 		Msg("Loading ACLs")
 
-	switch filepath.Ext(path) {
-	case ".yml", ".yaml":
-		return LoadACLPolicyFromBytes(policyBytes, "yaml")
```

Comment:
Maybe we need to emit some sort of warning/fatal here too?

Reply:
I suppose it errors when it cant unmarshal, but maybe we need a good error message.

Reply:
It would make more sense as a consumer to say that it's no longer supported instead of just spewing out unmarshaling errors.

Reply:
Thanks. Makes sense to throw a meaningful error. I've moved the errors in `LoadACLPolicyFromBytes`.